base/applications/notepad/lang/tr-TR.rc

```diff
@@ -173,12 +174,12 @@ BEGIN
     STRING_LINE_COLUMN "%d. Satır, %d. Sütun"
     STRING_PRINTERROR """%s"" dosyası yazdırılamıyor.\n\nYazıcının açık olduğundan ve doğru yapılandırıldığından emin olunuz."
     STRING_DEFAULTFONT "Lucida Console"
-    STRING_LINE_NUMBER_OUT_OF_RANGE "The specified line number is out of range."
-    STRING_NOWPRINTING "Now printing page..."
-    STRING_PRINTCANCELING "The print job is being canceled..."
-    STRING_PRINTCOMPLETE "Printing is successfully done."
-    STRING_PRINTCANCELED "Printing has been canceled."
-    STRING_PRINTFAILED "Printing failed."
+    STRING_LINE_NUMBER_OUT_OF_RANGE = "Belirtilen satır numarası aralık dışındadır.";
+    STRING_NOWPRINTING "Yazdırılıyor...";
+    STRING_PRINTCANCELING "Yazdırma işi iptal ediliyor...";
+    STRING_PRINTCOMPLETE "Yazdırma işlemi başarıyla tamamlandı.";
+    STRING_PRINTCANCELED "Yazdırma işlemi iptal edildi.";
+    STRING_PRINTFAILED "Yazdırma başarısız oldu.";
```

Comment:
Your syntax is not correct for RC file (using equals sign and semicolon), please undo these changes.